aws-athena/views/qc-vw_iasworld_sale_unmatched_sale_prices_sales_mydec.sql

```diff
@@ -0,0 +1,34 @@
+WITH sales_cte AS (
+    SELECT
+        SUBSTR(saledt, 1, 4) AS year,
+        price,
+        REGEXP_REPLACE(instruno, '[^0-9]', '') AS instruno
+    FROM iasworld.sales
+    WHERE
+        deactivat IS NULL
+        AND cur = 'Y'
+        AND SUBSTR(saledt, 1, 4) >= '2014'
+        AND CAST(SUBSTR(saledt, 1, 4) AS INTEGER) >= 2014
```

Comment:
**[Nitpick, non-blocking]** It seems to me like these are two versions of the same check, is that right? Is there a particular edge case in which we would need both?

Reply:
No these are different. For example, the query:
SELECT line_11_full_consideration FROM "sale"."mydec" 
WHERE document_number = '1406319113' 
results in a sale price of 50,000

while 
SELECT sale_price FROM "default"."vw_pin_sale" 
WHERE doc_no = '1406319113'

results in 0.

This tests matched document numbers to identify when prices are different.